projects/exui-common-lib/src/lib/components/loading-spinner/loading-spinner.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="loading-spinner__overlay" [ngStyle]="{'z-index': zIndex}">
+  <div class="loading-spinner__box" [ngStyle]="{'z-index': zIndex + 1}">
+    <div class="loading-spinner__fan">
+      <div></div>
```

Comment:
Could reduce this by replacing all the empty divs with <div *ngFor="let i of [].constructor(12)"></div>